ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/TransactionService.scala

```diff
@@ -627,15 +688,16 @@ class TransactionService(session: LedgerSession) extends LedgerTestSuite(session
       "Accept exercising a well-authorized multi-actor choice with coinciding controllers") {
       context =>
         for {
-          ledger <- context.participant()
-          Vector(operator, giver) <- ledger.allocateParties(2)
-          agreementFactory <- ledger.create(giver, AgreementFactory(giver, giver))
-          agreement <- ledger.exerciseAndGetContract[Agreement](
+          Vector(alpha, beta) <- context.participants(2)
+          operator <- alpha.allocateParty()
+          giver <- beta.allocateParty()
+          agreementFactory <- beta.create(giver, AgreementFactory(giver, giver))
+          agreement <- beta.exerciseAndGetContract[Agreement](
             giver,
             agreementFactory.exerciseAgreementFactoryAccept)
           triProposalTemplate = TriProposal(operator, giver, giver)
-          triProposal <- ledger.create(operator, triProposalTemplate)
-          tree <- ledger.exercise(giver, agreement.exerciseAcceptTriProposal(_, triProposal))
+          triProposal <- alpha.create(operator, triProposalTemplate)
+          tree <- beta.exercise(giver, agreement.exerciseAcceptTriProposal(_, triProposal))
```

Comment:
Same argument as above regarding flakiness between `alpha` and `beta`.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/2900/commits/0fa14d151df5a99067c88f4ce5ce11896b379e5a